assets/scss/templates/_main.scss

```diff
@@ -284,13 +286,12 @@ a.post-title {
   pre {
     display: block;
     padding: 9.5px;
-    margin: 10px 0px 10px;
+    margin-bottom: 0; // to remote the margin bottom from `.content *`
```

Comment:
```suggestion
    margin-bottom: 0; // to remove the margin bottom from `.content *`
```

Reply:
Also, why do we keep this code inside `_main.scss` if you created the `_code.scss` file?